lms/envs/aws.py

```diff
@@ -669,6 +669,7 @@
     FIELD_OVERRIDE_PROVIDERS += (
         'ccx.overrides.CustomCoursesForEdxOverrideProvider',
     )
+CCX_MAX_STUDENTS_ALLOWED = ENV_TOKENS.get('CCX_MAX_STUDENTS_ALLOWED', CCX_MAX_STUDENTS_ALLOWED)
```

Comment:
I see some value about having this value configurable, but I have not strong arguments about it. @pdpinch, what do you think?